src/main/webapp/index.html

```diff
@@ -70,7 +70,9 @@ <h2 class="masthead-subheading mb-0" style="text-align: left">A Student Mentor M
         <div class="col-lg-6 order-lg-1">
           <div class="p-5">
             <h2 class="display-4">Match with Mentors in the Tech Industry</h2>
-            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quod aliquid, mollitia odio veniam sit iste esse assumenda amet aperiam exercitationem, ea animi blanditiis recusandae! Ratione voluptatum molestiae adipisci, beatae obcaecati.</p>
+            <p>There are so many specialties in the computer science field that it can feel overwhelming for college students and interns to know what they want to do. 
```

Comment:
nit: Consider starting this paragraph with a stronger opening sentence. A variation of the last sentence could work well here: "Techish matches college students and interns with mentors who can teach you about their field." 

Also consider answering the what, why, and how for each of these paragraphs (see https://developers.google.com/tech-writing/one/paragraphs#answer_what_why_and_how)

Reply:
Good point, thanks!